internal/cli/cmd/clustermesh.go

```diff
@@ -162,6 +162,7 @@ func newCmdClusterMeshStatus() *cobra.Command {
 	cmd.Flags().BoolVar(&params.Wait, "wait", false, "Wait until status is successful")
 	cmd.Flags().DurationVar(&params.WaitDuration, "wait-duration", 15*time.Minute, "Maximum time to wait")
 	cmd.Flags().BoolVar(&params.SkipServiceCheck, "skip-service-check", false, "Do not require service IP of remote cluster to be available")
+	cmd.Flags().StringVarP(&params.Output, "output", "o", "summary", "Output format. One of: (json, summary).")
```

Comment:
Nit: As mentioned before, users need not specify "summary" output as you have already set it as default. Take a look at the above flag if you want a reference - 

`cmd.Flags().DurationVar(&params.WaitDuration, "wait-duration", 15*time.Minute, "Maximum time to wait")`